### DISCUSSION TITLE
browser not fully propagating

### DISCUSSION THREAD

**Discussion by dfengineer** [datalust/seq-tickets#1678] [General]
Installation has been tested on multiple systems with the same outcome. Current setup is using port 443 and a cert that is approved for the environment, and Windows Firewall disabled. IE is the only browser that works partially (see Pic, Seq at top left) and it doesn't work on Chrome, Edge or Firefox (chrome displayed in Pic). 

![image](https://user-images.githubusercontent.com/116911564/198702607-3f208d7a-0710-42c9-a5f9-f178274126e4.png)

**Comment by dfengineer** [comment]
Update, it only doesn't work using SSL. Must be a Chrome setting.

**Reply by nblumhardt** [reply]
Hi! Thanks for reaching out.

`https://localhost` won't work unless the name on the SSL certificate is `localhost`, and you used the name `localhost` in the `seq bind-ssl` command (or setup wizard).

If you need to test via the actual hostname but don't have DNS set up yet, you can [add an entry to your Windows `hosts` file](https://en.wikipedia.org/wiki/Hosts_(file)#File_content) pointing to 127.0.0.1 temporarily so that the name (e.g. `example.com`) resolves to localhost.

Hope this helps!
Nick

**Reply by dfengineer** [reply]
fixed the cert. same issue.

![image](https://user-images.githubusercontent.com/116911564/199060794-e8ba74b8-c52b-494e-9b5d-d39df5f97480.png)
